web/translation/translate.en_US.toml

```diff
@@ -312,16 +312,16 @@
 "advancedTemplate" = "Advanced"
 "generalConfigs" = "General Strategy"
 "generalConfigsDesc" = "These options will determine general strategy adjustments."
-"logConfigs" = "Log"
-"logConfigsDesc" = "Logs may affect your server's efficiency. It is recommended to enable it wisely only in case of your needs"
+"logConfigs" = "Log Settings"
+"logConfigsDesc" = "Enabling log may impact your machine's performance. It is recommended to enable it wisely only when necessary."
```

Comment:
It is a lot specific.
Using `machine` instead of `server` is not professional.
I prefer `efficiency` instead of `performance`. Machine prformance will not affect actually.